pkg/v1/cmd.go

```diff
@@ -603,41 +603,40 @@ func applyConfig(ctx context.Context, logger *logrus.Entry, org, repo, branch, d
 		}
 	}
 
-	extraVendor := map[string][]string{
-		"operator-controller": {"testdata/push", "testdata/registry", "hack/ci/custom-linters/analyzers/testdata"},
+	vendorDirs := []string{}
+	finder := internal.WithEnv(internal.WithDir(exec.CommandContext(ctx,
+		"find", ".", "-name", "go.mod",
+	), dir), os.Environ()...)
+	output, err := internal.RunCommand(logger, finder)
+	if err != nil {
+		return err
 	}
-
-	generatedPatches := []*exec.Cmd{
-		internal.WithEnv(internal.WithDir(exec.CommandContext(ctx,
-			"go", "mod", "tidy",
-		), dir), os.Environ()...),
-		internal.WithEnv(internal.WithDir(exec.CommandContext(ctx,
-			"go", "mod", "vendor",
-		), dir), os.Environ()...),
-		internal.WithEnv(internal.WithDir(exec.CommandContext(ctx,
-			"go", "mod", "verify",
-		), dir), os.Environ()...),
+	// Remove ".bingo" dirs
+	for _, d := range strings.Split(output, "\n") {
```

Comment:
I don't know if this is designed to be able to run on windows, but I don't think it would work properly due to the endline differences and you would have to do something like
```golang
strings.Split(strings.ReplaceAll(output, "\r\n", "\n"), "\n")
```

Reply:
This is only meant to run as a period job in Linux.